Header header
float32 min_threshold
float32 r_0 
float32 d_0

